
public class CFinalExcep {
	public static void main(String[] args) {
		int x=5;
		for (int y=2; y>=0; y=y-1) {
			try {
				int z=(x-1)/(y-1);
				System.out.println("AAAAA");
				if (y==0) return;
				// if (y==0) System.exit(0);
			}
			catch (Exception e) {
				System.out.println("BBBBB");
			}
			finally
			{
				System.out.println("CCCCC");
			}
		}
	}

}
